EPNTAP.tex

```diff
@@ -2501,8 +2504,8 @@ \subsection{Registering EPN Data Collections and Services}
 exposed through protocols different from TAP.  Normally, however, the
 tableset will be contained in a VODataService \xmlel{CatalogService}
 record with a TAP capability, and this capability will be an auxiliary
-capability as per DDC \citep{2019ivoa.spec.0520D}.  For one-table
-services a full TAPRegExt \citep{2012ivoa.spec.0827D} capability is also
+capability as per DDC \citep{2019ivoa.rept.0520D}.  For one-table
```

Comment:
The new ID here, `2019ivoa.rept.0520D`, is no longer defined.  I don't know whether that means the edit was wrong or an ivoatex bib update is required.